aks-node-controller/proto/aksnodeconfig/v1/kubelet_config.proto

```diff
@@ -38,6 +40,7 @@ message KubeletConfig {
 message Taint {
   string key = 1;
   string effect = 2;
+  string value = 3;
```

Comment:
I see you add the property, but I don't see any usage of it.
How does it work?

Reply:
This is part of `taint`. It's not being used now. My intention was to reserve it for future use. But it's not a must

Reply:
why do you need to reserve something if we're not using it?

Reply:
My original plan was to implement this by extracting taints out from kubelet config. But it seems it will need more time than I have expected. I will remove it from the contract. :)

Reply:
The linter will complain. Do you know if there is any way to suppress it?
```
proto/aksnodeconfig/v1/kubelet_config.proto:6:1:Previously present field "4" with name "startup_taints" on message "KubeletConfig" was deleted without reserving the name "startup_taints".
proto/aksnodeconfig/v1/kubelet_config.proto:6:1:Previously present field "1" with name "taints" on message "KubeletConfig" was deleted without reserving the number "1".
proto/aksnodeconfig/v1/kubelet_config.proto:6:1:Previously present field "4" with name "startup_taints" on message "KubeletConfig" was deleted without reserving the number "4".
```

Reply:
I think since it's not enforced in the PR, so I can still push the changes. Once it's in production, we will avoid doing this.

Reply:
Actually it's enforced.
![image](https://github.com/user-attachments/assets/5ef63b65-b6ae-4568-9edb-ca1c0f77b1c9)

Reply:
probably can use someone with superpower to force-merge the PR?

Reply:
I think it's safe in this case

Reply:
sounds good. If you approve it, I can try the superpower

Reply:
another note, it seems like CI changes are reacting on breaking changes in .proto

I think it should be possible to change it to react only to breaking change in .json format.